src/CLRunner.ts

```diff
@@ -140,6 +143,27 @@ export class CLRunner {
         return newRunner;
     }
 
+    private static validiatedModelOptions(modelOptions?: Partial<ModelOptions.CLModelOptions>): ModelOptions.CLModelOptions {
+        const sessionTimout = (!modelOptions
+            || !modelOptions.sessionTimout
+            || typeof modelOptions.sessionTimout !== 'number')
+            ? ModelOptions.DEFAULT_MAX_SESSION_LENGTH
+            : modelOptions.sessionTimout
+
+        const repromptThreshold = (!modelOptions
+            || !modelOptions.repromptThreshold
```

Comment:
The negation check is redundant with the `typeof` check.  If it's not a "number" is more constraining than !x